src/libraries/Common/src/Interop/Windows/Advapi32/Interop.TRUSTED_POSIX_OFFSET_INFO.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Advapi32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        internal sealed class TRUSTED_POSIX_OFFSET_INFO
```

Comment:
```suggestion
        internal struct TRUSTED_POSIX_OFFSET_INFO
```

Reply:
Done in 25b4d8e237d7d3b0ae195d0ae7bc8bcd8c87dcce